src/js/managers/storageManager.js

```diff
@@ -0,0 +1,33 @@
+import { ERROR_MESSAGE, STORE } from '../utils/constants.js';
+
+const storageManager = {
+  keys: {
+    videoId: 'videoId',
+  },
+
+  storeVideoId: function (videoId) {
+    if (!this.hasVideoID(videoId)) {
+      const videoList = this.getVideoIdList();
+      videoList.push(videoId);
+      localStorage.setItem(this.keys.videoId, JSON.stringify(videoList));
+    }
+  },
+
+  getVideoIdList: function () {
+    const videoList = JSON.parse(localStorage.getItem(this.keys.videoId));
+    return videoList || [];
+  },
+
+  hasVideoID: function (videoId) {
+    const videoList = this.getVideoIdList();
+    return videoList ? videoList.includes(videoId) : false;
```

Comment:
변수명은 언제나 명확한 게 좋은 거 같습니다ㅎㅎ,
```suggestion
    const videoIdList = this.getVideoIdList();
    return videoIdList ? videoIdList.includes(videoId) : false;
```

Reply:
넵! 수정했습니다 🙂